src/test/java/com/wikia/webdriver/testcases/specialpagestests/UserAndRights.java

```diff
@@ -47,6 +50,28 @@ public void blockedUserShouldSeeMessageOnArticleEdit() {
     edit.verifyBlockedUserMessage();
   }
 
+  @Test(groups = {"usersAndRights002", "UsersAndRights"}, dependsOnMethods = {
+      "staffCanBlockUser"})
+  @Execute(asUser = User.BLOCKED_USER)
+  public void blockedUserShouldBeAbleToChangeEmail() {
+      final String newEmailAddress = "myAwesomeEmail@email.co.uk";
```

Comment:
This is a local final field, It's here so no other guy can modify it after it's initialized. Moving it to class field is a horrible idea.